### DISCUSSION TITLE
why not super resolution models, such as SRGAN, ESRGAN

### DISCUSSION THREAD

**Discussion by ucasyjz** [open-mmlab/mmgeneration#464] [How-to] [answered]

**Comment by plyfager** [comment, marked as answer]
Hi @yang326922943, You can get these models at https://github.com/open-mmlab/mmediting.

**Comment by ucasyjz** [comment]
Thanks

**Comment by ucasyjz** [comment]
here I have another question, can I incorporate the super resolution model into 'mmdetection',
